README.md

```diff
@@ -33,6 +33,20 @@ julia> serve() # starts the local server & the file watching
 
 Open a Browser and go to `http://localhost:8000/` to see the content being rendered; try modifying files (e.g. `index.html`) and watch the changes being rendered immediately in the browser.
 
+```julia-repl
+julia> using LiveServer
+julia> serve(host="0.0.0.0", port=8001, dir=".") # starts the remote server & the file watching
+✓ LiveServer listening on http://0.0.0.0:8001...
+  (use CTRL+C to shut down)
+```
+```julia-repl
+julia -e 'using LiveServer; serve(host="0.0.0.0", port=8001, dir=".")'
+such as: 
+python3 -m http.server 8001
```

Comment:
L44-L45 should be in the comment rather than in the code block, optionally you can just take it out and clarify:

In the REPL: (first block). In the terminal: (second block)